packages/slugify/build.sh

```diff
@@ -0,0 +1,21 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/benlinton/slugify
+TERMUX_PKG_DESCRIPTION="Bash command that converts filenames and directories to a web friendly format."
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=1.0.1
+TERMUX_PKG_SRCURL=$TERMUX_PKG_HOMEPAGE/archive/v$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=f6873b062119d3eaa7d89254fc6e241debf074da02e3189f12e08b372af096e5
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_DEPENDS=bash
+
+termux_step_make_install() {
+	cd "$TERMUX_PKG_SRCDIR"
+	install -D slugify -t "$TERMUX_PREFIX/usr/bin"
+	install -D slugify.1 -t "$TERMUX_PREFIX/usr/share/man/man1"
```

Comment:
These directories should be `$TERMUX_PREFIX/bin` and `$TERMUX_PREFIX/share/man/man1` respectively.